src/Framework/TestCase.php

```diff
@@ -174,6 +180,10 @@ protected function tearDown()
 	{
 	}
 
+	/** @internal */
+	public function failsafeTearDown() {
```

Comment:
Name it as `_tearDown()` and fix coding style. Double empty line between methods, one empty line after the last one, curly bracket on separated line.